cdap-data-fabric/src/main/java/co/cask/cdap/store/StoreDefinition.java

```diff
@@ -16,21 +16,29 @@
 
 package co.cask.cdap.store;
 
+import co.cask.cdap.common.conf.Constants;
+import co.cask.cdap.proto.id.NamespaceId;
 import co.cask.cdap.spi.data.StructuredTableAdmin;
 import co.cask.cdap.spi.data.TableAlreadyExistsException;
 import co.cask.cdap.spi.data.table.StructuredTableId;
 import co.cask.cdap.spi.data.table.StructuredTableRegistry;
 import co.cask.cdap.spi.data.table.StructuredTableSpecification;
 import co.cask.cdap.spi.data.table.field.Fields;
+import com.google.common.collect.ImmutableMap;
 
 import java.io.IOException;
+import java.util.Map;
 
 /**
  * A class which contains all the store definition, the table name the store will use, the schema of the table should
  * all be specified here.
  * TODO: CDAP-14674 Make sure all the store definition goes here.
  */
 public final class StoreDefinition {
+  public static final Map<String, String> METRICS_TAGS = ImmutableMap.of(
+    Constants.Metrics.Tag.COMPONENT, Constants.Service.DATASET_MANAGER,
```

Comment:
Would be good to use a new component name for system datastore metrics.

Reply:
Moved to constant